### PR TITLE
[WF-672] Inverted Card

### PR DESCRIPTION
**Pull request by blurbyte**
# Inverted Card

### 🚀 Major Changes

-  Added `inverted` Card variant.
  - Implemented `Card.HeadstoneAvatar` subcomponent, which should improve usability of the `Card` component, so user don't have to worry about setting proper color variant and size of `Avatar`. It's still possible to use custom components for headstone.

## Screenshot(s):
Screenshot 2022-11-01 at 13 51 12

### CODE REVIEW

File: src/card/styles.ts [hunk 1]
```diff
@@ -22,9 +24,11 @@ export function cardStyle({
     css`
       padding-top: 28px;
     `}
-    background-color: ${tokens.colors.white};
+    background-color: ${inverted
+      ? tokens.colors.navyLight
+      : tokens.colors.white};
     border: ${tokens.borderWidth.thin} solid
-      ${hexToRgba(tokens.colors.navy, 0.15)};
+      ${hexToRgba(inverted ? tokens.colors.white : tokens.colors.navy, 0.15)};
```
Comment (ixTec) [inline src/card/styles.ts:31]: Do we also want to have a different coloured box shadow on hover for inverted too?

**Review by ixTec — CHANGES_REQUESTED** [review]
Few minor comments, but otherwise looks good!

File: src/card/styles.ts (hunk 1 above)
Comment (blurbyte) [reply src/card/styles.ts:31]: Not according to designs. Not sure how to make it visible (besides making it light grey, which will look weird).

File: src/card/headstone-item.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Avatar } from '../avatar';
+
+type HeadstoneItemProps = {
+  /* Content of the item. In general, pass Waffles [Logomark asset](/components/asset/#logomark-assets) or a string containing only one character. */
+  content: JSX.Element | string;
+  /* [skip docs] */
+  inverted?: boolean;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+function HeadstoneItem({
```
[the remark this answers is not in the document]
Comment (blurbyte) [reply src/card/headstone-item.tsx:10]: Good idea.

**Review by ixTec — APPROVED** [review]

**Pull request merged**